src/MongoDB/Driver/Manager.cpp

```diff
@@ -466,12 +466,22 @@ const StaticString
 Array HHVM_METHOD(MongoDBDriverManager, __debugInfo)
 {
 	MongoDBDriverManagerData* data = Native::data<MongoDBDriverManagerData>(this_);
+	size_t                        i;
+	mongoc_server_description_t **sds;
+	size_t                        n;
```

Comment:
Same as https://github.com/mongodb/mongo-hhvm-driver/pull/78/files#r57771157. Should everything be aligned here, or just forgo it?